### DISCUSSION TITLE
Configuring the initial setup ports/directory?

### DISCUSSION THREAD

**Discussion by Mazz33** [rtCamp/Frappe-Manager#224] [Q&A] [answered]
Hello,

I am wondering if there's a built in way to change the first installation configurations, for example instead of ~/frappe, to choose a different directory like ~/Desktop/frappe-test. And if it's possible to change the nginx  ports to not use 80/443.

I want to set up new sites to use ports 8080/8443 instead, editing the nginx compose file does make it use those ports, but the sites are configured to use port 80/443 i believe?

I couldn't find in Wiki if it's possible to choose where to install, and what ports to use.

Thank you.

**Comment by Xieyt** [comment, marked as answer]
Currently, there is no built in way to have port change or directory change. Feature request issue is always welcome.

Changing directory is not possible in any way as of now, but changing ports is possible and can be done like this
- Open `~/frappe/services/docker-compose.yml` and change ports like this

```bash
  global-nginx-proxy:
    container_name: fm_global-nginx-proxy
     image: jwilder/nginx-proxy
     ports:
-    - 80:80
-    - 443:443
+    - 8080:80
+    - 8443:443
```
- Then run this command to reflect changes `docker compose -f ~/frappe/services/docker-compose.yml up -d`
